docs(ai-agents): Add AI connector tutorials and Agent engine positioning (#70461)
airbytehq/airbyte: docs/ai-agents/connectors/tutorial-mcp.md

---
sidebar_label: "Connector MCP tutorial"
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> # Get started with AI connectors: Connector MCP

Reviewer: ```suggestion
# Get started with Agent connectors: Connector MCP
```

## Overview

This tutorial is for AI engineers and other technical users who work with data and AIs. It assumes you have basic knowledge of the following.

- Claude Code or the AI agent of your choice
- MCP servers
- Stripe, or a different third-party service you want to connect to

## Before you start

Before you begin this tutorial, ensure you have installed the following software.

- Claude Code or the agent of your choice, and the plan necessary to run it locally
- [Python](https://www.python.org/downloads/) version 3.13.7 or later
- [uv](https://github.com/astral-sh/uv)
- An account with Stripe, or a different third-party [supported by AI connectors](https://github.com/airbytehq/airbyte-agent-connectors/tree/main/connectors).

## Part 1: Clone the Connector MCP repository

Clone the Connector MCP repository.

```bash
git clone https://github.com/airbytehq/connector-mcp/
```

Once git finishes cloning, change directory into your repo.

```bash
cd connector-mcp
```

## Part 2: Configure the connector you want to use

### Create a connector configuration file

> The `configured_connectors.yaml` file defines which AI connectors you are making available through the MCP and which secrets you need for authentication.

Reviewer: ```suggestion
The `configured_connectors.yaml` file defines which Agent connectors you are making available through the MCP and which secrets you need for authentication.
```

1. Create a file called `configured_connectors.yaml`. It's easiest to add this file to the root, but if you want to add it somewhere else, you can instruct the MCP where to find it later.

> 2. Add your connector definition to this file. The `connector_name` field specifies which connector to load from the [Airbyte AI Connectors registry](https://github.com/airbytehq/airbyte-agent-connectors). The keys under `secrets` are logical names that must match environment variables in your `.env` file.

Author: ```suggestion
2. Add your connector definition to this file. The `connector_name` field specifies which connector to load from the [Airbyte AI Connectors registry](https://connectors.airbyte.ai/registry.json). The keys under `secrets` are logical names that must match environment variables in your `.env` file.
```
